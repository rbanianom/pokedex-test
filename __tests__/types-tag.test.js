import React from 'react';
import renderer from 'react-test-renderer';
import TypesTag from '../helper/types-tag';

const typeList = [
  {
    slot: 1,
    type: {
      name: 'grass',
      url: 'https://pokeapi.co/api/v2/type/12/',
    },
  },
  {
    slot: 2,
    type: {
      name: 'poison',
      url: 'https://pokeapi.co/api/v2/type/4/',
    },
  },
];

it('tag type render correctly', () => {
  const tree = renderer.create(<TypesTag types={typeList} />).toJSON();

  expect(tree).toMatchSnapshot();
});
