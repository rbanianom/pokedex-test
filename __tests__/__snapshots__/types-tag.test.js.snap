// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag type render correctly 1`] = `
Array [
  <span
    className="ant-tag ant-tag-has-color"
    style={
      Object {
        "backgroundColor": "#72c134",
      }
    }
  >
    Grass
  </span>,
  <span
    className="ant-tag ant-tag-has-color"
    style={
      Object {
        "backgroundColor": "#904392",
      }
    }
  >
    Poison
  </span>,
]
`;
